examples/http/nginx_status.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * this driver is based on nginx module "ngx_http_stub_status_module"
+ * You need to read this http://nginx.org/en/docs/http/ngx_http_stub_status_module.html 
+ * before using this driver
+ * this driver is tested under nginx version: nginx/1.14.0 (Ubuntu)
+ */
+// nginx status page config
+var nginxStatusHttpConfig = {
+    port: 10011,
+    url: "/nginx_status"
```

Comment:
My installation uses "/basic_status" which is also inline with the  doc URL you have added on the comments above:
http://nginx.org/en/docs/http/ngx_http_stub_status_module.html